libs/transactions-journey/feature/src/lib/components/transaction-item/transaction-item.component.html

```diff
@@ -9,9 +9,16 @@
   transaction.bookingDate | date : 'MMM. d'
 }}</span>
 <span class="transactions-item__description">
-  <span class="transactions-item__counterParty">{{
-    transaction.counterPartyName || transaction.merchant?.name
-  }}</span>
+  <span class="transactions-item__counterParty"
+    >{{ transaction.counterPartyName || transaction.merchant?.name }}
+    <img
+      src="./assets/transactions-journey/{{ iconName }}"
```

Comment:
@manisha-backbase Can we have a variable to the assets folder or this hardcoded value will always work for local, staging and production?

Reply:
The reference should work in all the environemnts